pkg/docker/docker.go

```diff
@@ -463,20 +463,21 @@ func getScriptsURL(image *docker.Image) string {
 	if len(scriptsURL) == 0 {
 		scriptsURL = getLabel(image, "io.s2i.scripts-url")
 		if len(scriptsURL) > 0 {
-			glog.V(0).Infof("warning: The 'io.s2i.scripts-url' label is deprecated. Use %q instead.", ScriptsURLLabel)
+			glog.V(0).Infof("warning: Image %s uses deprecated label 'io.s2i.scripts-url', please migrate it to %s instead!",
+				image.ID, ScriptsURLLabel)
```

Comment:
Here we also include `image.ID` into the message. Unfortunately it will be SHA1 sum of the image, because on that level we don't have a human readable name, but only object with image data.

BTW it will be especially useful when we'll have not only builder image but also a runtime image.